### DISCUSSION TITLE
评论区登录后无法退出

### DISCUSSION THREAD

**Discussion by abovewave** [walinejs/waline#1126] [Q&A] [answered]
登录后昵称邮箱网址栏直接消失了，请问该如何退出登录呢？我点击昵称后进入的页面里即使退出，回到博客本身依旧是登录状态……

在评论管理系统里也无法注销账号
<img width="801" alt="Screen Shot 2022-06-16 at 23 36 43" src="https://user-images.githubusercontent.com/73049366/174226433-81355d9e-a343-4132-8fd0-5c95964c0187.png">

**Comment by lizheming** [comment, marked as answer]
头像右上角 hover 有个 x 按钮。这个应该是你博客本身额样式冲突问题，可以自行解决一下。

**Reply by abovewave** [reply]
看到了谢谢！
